src/chord/node.cljs

```diff
@@ -0,0 +1,42 @@
+(ns chord.node
+  (:require [cljs.nodejs :as nodejs]
+            [chord.channels :as channels]
+            [chord.format :as format]))
+
+(defn ws-server [ws-opts {:keys [read-ch write-ch :as chord-opts]} handler]
```

Comment:
maybe a more consistent signature here is `(defn ws-server [handler {:keys [read-ch write-ch ws-opts]}]...)` ? (looking at `wrap-websocket-handler`, above)